Diagnostics/HealthChecker/Analyzer/Invoke-AnalyzerEngine.ps1

```diff
@@ -1548,16 +1548,16 @@ Function Invoke-AnalyzerEngine {
     }
 
     $additionalDisplayValue = [string]::Empty
-    $smb1Status = $osInformation.Smb1ServerSettings.Smb1Status
+    $smb1Settings = $osInformation.Smb1ServerSettings
 
     if ($osInformation.BuildInformation.MajorVersion -gt [HealthChecker.OSServerVersion]::Windows2012) {
         $displayValue = "False"
         $writeType = "Green"
 
-        if ($smb1Status -band 1) {
+        if (-not ($smb1Settings.SuccessfulGetInstall)) {
             $displayValue = "Failed to get install status"
             $writeType = "Yellow"
-        } elseif ($smb1Status -band 2) {
+        } elseif ($smb1Settings.Installed) {
```

Comment:
`$smb1Settings.Installed` doesn't work. Must be: `$smb1settings.WindowsFeature.Installed`